packages/grid/x-data-grid-pro/src/utils/tree/utils.ts

```diff
@@ -34,8 +34,8 @@ export const getNodePathInTree = ({
   let node = tree[id] as GridGroupNode | GridLeafNode;
 
   while (node.id !== GRID_ROOT_GROUP_ID) {
-    path.push({
-      field: (node as GridGroupNode).groupingField,
+    path.unshift({
```

Comment:
On second thought, I'm going to keep `.push` and just reverse the array later, because it looks like `unshift` is ~70% slower even for short arrays: https://jsperf.app/wisoqe
<img width="981" src="https://user-images.githubusercontent.com/13808724/235723988-d7c9bfc2-eaa2-489c-b863-391643d012b5.png">